src/core/tsi/ssl_transport_security.cc

```diff
@@ -1048,9 +1048,11 @@ static tsi_result ssl_handshaker_result_extract_peer(
                                    &alpn_selected_len);
   }
 
+  STACK_OF(X509) *peer_chain = SSL_get0_verified_chain(impl->ssl);
```

Comment:
Do we still get an unverified certificate chain if client or server does not validate the peer's certificate during the handshake?

Reply:
Yes